src/FormParseNode.cs

```diff
@@ -208,7 +237,8 @@ public IEnumerable<T> GetCollectionOfPrimitiveValues<T>()
     {
         if(string.IsNullOrEmpty(rawValue))
             return null;
-        if(typeof(T).GetCustomAttributes<FlagsAttribute>().Any())
+
+        if (typeof(T).GetCustomAttributes<FlagsAttribute>().GetEnumerator().MoveNext())
```

Comment:
I thought about doing this. The catch is that the enumerator may be `IDisposable`. I didn't actually verify if it is or not but the `Any()` pattern handles it.

Reply:
good catch!